monasca_persister/README.rst

```diff
@@ -46,45 +46,63 @@ Those data points are mapped to NGSI Entities according to the following:
      - Monasca measurement
      - Monasca value_meta
      - NGSI attributes
-   * - region.allocated_ip
+   * - region.info
      - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
-     - region.allocated_ip
+       | {component}_version
+     - region.info
      - |
-       |
-       | *[none, depending on configuration]* (\*)
-     - ipAvailable
-   * - region.pool_ip
-     - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
-     - region.pool_ip
+       | {component}_version
      - |
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
-     - | ipTot
+       | {component}_version
+   * - region.allocated_ip
+     - | name
+       | location
+       | latitude
+       | longitude
+       | ram_allocation_ratio
+       | cpu_allocation_ratio
+       | {component}_version
+     - region.allocated_ip
+     - |
+       |
+       | *[none, depending on configuration]* (\*)
+     - ipAvailable
+   * - region.pool_ip
+     - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
+       | {component}_version
+     - region.pool_ip
+     - |
+       |
+       | *[none, depending on configuration]* (\*)
+     - | ipTot
    * - region.used_ip
      - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
+       | {component}_version
      - region.used_ip
      - |
        |
```

Comment:
I would like to add in the documentarion that the {component}_version will follow the data in [http://releases.openstack.org/].

Hopefully I hope...